### ISSUE TITLE
The Broccoli Plugin: [BroccoliTypeScriptCompiler] failed with:

### ISSUE THREAD

**Issue by jboss42** [angular/angular-cli#836]
Error: EPERM: operation not permitted, symlink 'F:\work\angular-starter\tmp\broccoli_type_script_compiler-cache_path-bHtZ3JSG.tmp\src\app\angular-starter.component.spec.js.map' -> 'F:\work\angular-starter\tmp\broccoli_type_script_compiler-output_path-Fpg8hL5D.tmp\src\app\angular-starter.component.spec.js.map'
    at Error (native)
    at Object.fs.symlinkSync (fs.js:897:18)
    at BroccoliTypeScriptCompiler._outputFile (F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\broccoli-typescript.js:206:8)
    at F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\broccoli-typescript.js:108:18
    at Array.forEach (native)
    at F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\broccoli-typescript.js:107:30
    at Array.forEach (native)
    at BroccoliTypeScriptCompiler._doIncrementalBuild (F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\broccoli-typescript.js:93:19)
    at BroccoliTypeScriptCompiler.build (F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\broccoli-typescript.js:43:10)
    at F:\work\angular-starter\node_modules\angular-cli\node_modules\broccoli-caching-writer\index.js:152:21
    at lib$rsvp$$internal$$tryCatch (F:\work\angular-starter\node_modules\angular-cli\node_modules\broccoli-caching-writer\node_modules\rsvp\dist\rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (F:\work\angular-starter\node_modules\angular-cli\node_modules\broccoli-caching-writer\node_modules\rsvp\dist\rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (F:\work\angular-starter\node_modules\angular-cli\node_modules\broccoli-caching-writer\node_modules\rsvp\dist\rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (F:\work\angular-starter\node_modules\angular-cli\node_modules\broccoli-caching-writer\node_modules\rsvp\dist\rsvp.js:1198:9)
    at nextTickCallbackWith0Args (node.js:420:9)
    at process._tickCallback (node.js:349:13)

The broccoli plugin was instantiated at: 
    at BroccoliTypeScriptCompiler.Plugin (F:\work\angular-starter\node_modules\angular-cli\node_modules\broccoli-caching-writer\node_modules\broccoli-plugin\index.js:10:31)
    at BroccoliTypeScriptCompiler.CachingWriter [as constructor] (F:\work\angular-starter\node_modules\angular-cli\node_modules\broccoli-caching-writer\index.js:21:10)
    at BroccoliTypeScriptCompiler (F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\broccoli-typescript.js:27:10)
    at Angular2App._getTsTree (F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\angular2-app.js:321:18)
    at Angular2App._buildTree (F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\angular2-app.js:116:23)
    at new Angular2App (F:\work\angular-starter\node_modules\angular-cli\lib\broccoli\angular2-app.js:53:23)
    at module.exports (F:\work\angular-starter\angular-cli-build.js:6:10)
    at Class.module.exports.Task.extend.setupBroccoliBuilder (F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\lib\models\builder.js:55:19)
    at Class.module.exports.Task.extend.init (F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\lib\models\builder.js:89:10)
    at new Class (F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\node_modules\core-object\core-object.js:18:12)
    at Class.module.exports.Task.extend.run (F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\lib\tasks\serve.js:15:19)
    at F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\lib\commands\serve.js:64:24
    at lib$rsvp$$internal$$tryCatch (F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (F:\work\angular-starter\node_modules\angular-cli\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1198:9)

**Comment by jboss42** [comment]
I created project using angular-cli.
when in prompt enter [ng serve], raised issue.

**Comment by JesseChrestler** [comment]
Try to run the command in administrator mode. This was an issue for me that was resolved when I ran it as an administrator.

**Comment by alexkibler** [comment]
Is there any way to run without administrator mode?  There is absolutely no way I can convince the infrastructure team to allow me to have administrator privileges.

**Comment by JesseChrestler** [comment]
@alexkibler Not until some adjustments are made to allow it. You can track the issue here #370

**Comment by filipesilva** [comment]
Admin mode is no longer needed, fixed in https://github.com/angular/angular-cli/pull/905.
